resources/tax_type/ca_sk_pst.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Canada - Saskatchewan",
+    "generic_label": "PST",
+    "display_inclusive": false,
+    "zone": "ca_sk_vat",
+    "tag": "CA",
+    "rates": [
+        {
+            "id": "ca_sk_pst_standard",
+            "name": "Standard",
+            "default": true,
+            "amounts": [
+                {
+                    "id": "ca_sk_pst_standard_2013",
+                    "amount": 0.05,
+                    "start_date": "2013-04-01"
```

Comment:
Can probably be excluded? It's been 0.05 since October 28, 2006 http://finance.gov.sk.ca/taxes/pst/